app/javascript/controllers/teams_controller.js

```diff
@@ -0,0 +1,55 @@
+import { Controller } from "@hotwired/stimulus"
+
+document.querySelectorAll('#addUserBtn, #addTeamBtn').forEach(function(button) {
```

Comment:
use kebab (ie - separated)  for naming ids, class etc, here this should be #add-user-btn.